Xwt.Gtk/Xwt.GtkBackend/TableViewBackend.cs

```diff
@@ -498,10 +499,14 @@ public void QueueDraw (object target, Gtk.TreeIter iter)
 	class CustomTreeView: Gtk.TreeView
 	{
 		WidgetBackend backend;
-		
+		TreePath delayedSelection;
+		TreeViewColumn delayedSelectionColumn;
+
 		public CustomTreeView (WidgetBackend b)
 		{
 			backend = b;
+			base.DragBegin += (_, __) =>
```

Comment:
can't we use an `OnDragBegin` override for this?

Reply:
Tried that, it doesn't get call for some reason :(

Reply:
Hhm, that's strange. Usually some handlers need to be attributed with `[GLib.ConnectBefore]`, but overrides should just work. But in this case it's ok to use the event then.